Frameworks/UIKit/UITapGestureRecognizer.mm

```diff
@@ -256,12 +256,12 @@ - (void)touchesEnded:(NSSet*)touches withEvent:(UIEvent*)event {
     }
 
     id allTouches = [event allTouches];
-    unsigned count = [touches count];
 
-    if (count > _numberOfTouchesRequired || EbrGetMediaTime() - _tapTime > 0.75f) {
-        TraceVerbose(TAG, L"too many touches, numberOfTouchesRequired=%d, count=%d", _numberOfTouchesRequired, count);
+    // We may have to revisit this logic, since the UITouch tap count should drive it.
```

Comment:
I'd rather not mess with it at this time. I'm not convinced our tapCount implementation is completely interoperable, and I'd rather we address the concerns of the task before investing in a reimplementation of something that may indeed be rewritten with WinRT gestures.